File: backend/src/main/java/com/example/demo/entities/BookingRequest.java
package com.example.demo.entities;

import java.sql.Date;
import java.time.LocalDateTime;
import java.util.Set;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.persistence.Table;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;


@Entity
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@ToString
@Table(name = "booking_requests")
public class BookingRequest {
	@Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "req_id")
    private int req_id;

    @ManyToOne
    @JoinColumn(name = "customer_id")
    private Customer customer;

    @ManyToOne
    @JoinColumn(name = "car_id")
    private Car car;

    @ManyToOne
    @JoinColumn(name = "package_id")
    private Package pack;

    @Column(name = "journey_date_time")
    //@Temporal(TemporalType.TIMESTAMP)
    private LocalDateTime journey_date_time;

    @Column(name = "status")
    private int status;
}

File: dot.net/BookMyCar/Models/BookingRequest.cs
using System;
using System.Collections.Generic;

namespace BookMyCar.Models;

public partial class BookingRequest
{
    public int ReqId { get; set; }

    public int? CustomerId { get; set; }

    public int? CarId { get; set; }

    public int? PackageId { get; set; }

    public DateTime? JourneyDateTime { get; set; }

    public int? Status { get; set; }

    public DateTime? ExpectedReturnDate { get; set; }

    public virtual ICollection<Booking> Bookings { get; set; } = new List<Booking>();

    public virtual Car? Car { get; set; }

    public virtual Customer? Customer { get; set; }

    public virtual Package? Package { get; set; }
}
